homeassistant/components/webostv/device_trigger.py

```diff
@@ -35,6 +46,22 @@ async def async_get_triggers(_hass: HomeAssistant, device_id: str) -> list[dict]
     return triggers
 
 
+async def async_validate_trigger_config(
+    hass: HomeAssistant, config: ConfigType
+) -> ConfigType:
+    """Validate config."""
+    config = TRIGGER_SCHEMA(config)
```

Comment:
I think the device automation framework will apply the trigger schema as long as it's defined. We don't need to do it manually.

Reply:
Hmm, actually it looks like it doesn't apply the schema if we define `async_validate_trigger_config`.

https://github.com/home-assistant/core/blob/18f4d125c39b1e6c33f8d1ccb36a5134c3b96e9e/homeassistant/helpers/trigger.py#L48-L51